### PR TITLE
OCM-4460 | test: automate id:OCP-73018 validations for breakglasscred

### PR DESCRIPTION
**Pull request by aaraj7**
output log:- https://privatebin.corp.redhat.com/?d0c20245dd399d9f#aHq5Gi3rNryVpAwQxSAYG3rSDGMCZqUfyujvfVnLL9o

### CODE REVIEW

File: tests/e2e/hcp_external_auth_test.go [hunk 1]
```diff
@@ -11,87 +11,187 @@ import (
...
+				// expiration timestamp changes from 4 to 5 seconds ahead, so have to remove second from time stamp
+				expirationTime, err := time.Parse("Jan _2 2006 15:04:05 MST", output.ExpireAt)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(output.ID).To(Equal(breakGlassCredID))
+				Expect(expirationTime.Format("Jan _2 2006 15:04 MST")).To(Equal(expiredTime))
+				Expect(output.Username).To(Equal(userName))
+				Expect(output.Status).To(Equal("issued"))
+
+				By("Get the issued credential")
+				_, err = rosaClient.BreakGlassCredential.GetIssuedCredential(clusterID, breakGlassCredID)
```
Comment (yingzhanredhat) [inline tests/e2e/hcp_external_auth_test.go:86]: Why not check the output for 'rosa list break-glass-credential'?

Comment (aaraj7) [reply tests/e2e/hcp_external_auth_test.go:86]: I have checked in line number 64

**Pull request merged**